ui/src/literature/components/ImprintInfo.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import InlineList, { SEPARATOR_AND } from '../../common/components/InlineList';
```

Comment:
Can you add some tests for this component?

Reply:
updated